docs/user/callbacks.rst

```diff
@@ -199,6 +199,26 @@ it is passed to the scoring function. The main reason why we need this
 is that sometimes, the target is not of a form expected by sklearn and
 we need to process it before passing it on.
 
+On top of the two described scoring callbacks, skorch also provides
+:class:`.PassthroughScoring`. This callback does not actually calculate
+any new scores. Instead it uses an existing score that is calculated
+for each batch and determines the average of this score, which is
```

Comment:
```suggestion
for each batch (the train loss, for example) and determines the average of this score, which is
```

Maybe this makes it easier to relate how and when this callback is useful?

Reply:
fixed